DetectorDescription/DDCMS/interface/DDSolidShapes.h

```diff
@@ -0,0 +1,80 @@
+#ifndef DETECTOR_DESCRIPTION_DDCMS_DD_SOLID_SHAPES_H
+#define DETECTOR_DESCRIPTION_DDCMS_DD_SOLID_SHAPES_H
+
+#include <iosfwd>
+
```

Comment:
@ianna given the header problems I would say that here at least 

```
#include <array>
#include <algorithm>
```

are needed (to be verified), do you agree?

Reply:
and 

```
#include <iterator>
```

as well

Reply:
thanks, @fabiocos - I'll fix it in the next PR